CHANGELOG.md

```diff
@@ -1,250 +1,310 @@
-| Fixed a bug that prevented the logo from loading when using `upward-php`                                        |  **Bugfix**  | [#1637][] |
+| Description                                                                                 | Change type  |    PR     |
+| :------------------------------------------------------------------------------------------ | :----------: | :-------: |
+| Added meta descriptions to root pages                                                       | **Feature**  | [#2035][] |
+| Added email validation to ForgotPasswordForm                                                | **Feature**  | [#1997][] |
+| Improved performance by lazy loading AuthModal                                              | **Feature**  | [#1955][] |
+| Improved performance by lazy loading Category filters                                       | **Feature**  | [#1971][] |
+| Improved Image fit with Fastly                                                              | **Feature**  | [#1976][] |
+| Added product page breadcrumbs                                                              | **Feature**  | [#1960][] |
+| Added category page breadcrumbs                                                             | **Feature**  | [#1949][] |
+| Improved product page caching                                                               | **Feature**  | [#1935][] |
+| Added PageBuilder components                                                                | **Feature**  | [#1872][] |
+| Implemented the `apollo-link-retry` Link                                                    | **Feature**  | [#1799][] |
+| Created component for showing categories with no products                                   | **Feature**  | [#1496][] |
+| Updated template HTML to include noscript fallback                                          |  **Update**  | [#2034][] |
+| Removed id from `getCustomer` response object                                               |  **Update**  | [#2028][] |
+| Added Email validation in signin form                                                       |  **Update**  | [#1981][] |
+| Added support for Carousel appearance for Product content type                              |  **Update**  | [#1951][] |
+| Updated static to venia-static                                                              |  **Update**  | [#1932][] |
+| Upgraded Apollo View layer to hooks                                                         |  **Update**  | [#1827][] |
+| Improved efficienct of the `getNavigationMenu` query                                        |  **Update**  | [#1867][] |
+| Added field labels for inputs                                                               |  **Update**  | [#1845][] |
+| Update mixin references to talon                                                            |  **Update**  | [#1820][] |
```

Comment:
Do we capitalize `Talon` since it is a proper noun? Is it a proper noun? I fail at grammar sometimes...